nft/collection/collection_register_encode.go

```diff
@@ -7,32 +7,77 @@ import (
 	"github.com/ProtoconNet/mitum2/base"
 	"github.com/ProtoconNet/mitum2/util"
 	"github.com/ProtoconNet/mitum2/util/encoder"
-	"github.com/ProtoconNet/mitum2/util/hint"
 )
 
-func (form *CollectionRegisterForm) unmarshal(
+// func (form *CollectionRegisterForm) unmarshal(
```

Comment:
How about removing this annotated code? Should we keep it?
Other (annotated) parts, too.

Reply:
@wyuinche I have removed it.